File.lua

```diff
@@ -393,6 +393,32 @@ function torch.load(filename, mode, referenced)
    return object
 end
 
+-- Loads binary files saved on a 64 bit OS
+function torch.load64(filename, referenced)
+   assert(referenced == nil or referenced == true or referenced == false, 'true or false (or nil) expected for referenced')
+   referenced = referenced == nil and true or referenced
+   local file = torch.DiskFile(filename, 'r')
+   file['binary'](file)
+   file:referenced(referenced)
+   file:longSize(8)
+   local object = file:readObject()
+   file:close()
+   return object
+end
+
+-- Loads binary files saved on a 32 bit OS
+function torch.load32(filename, referenced)
+   assert(referenced == nil or referenced == true or referenced == false, 'true or false (or nil) expected for referenced')
```

Comment:
Yes, absolutely right! Will do it!
What about
```lua
loadWithLongSize(filename, mode, references, size)
```
so that even `torch.load()` calls the same parametrised function?